ynh-dev

```diff
@@ -306,6 +308,52 @@ elif [ "$1" = "use-git" ]; then
     done
 
 
+elif [ "$1" = "test" ]; then
+    check_yunohost_vm
+    VERSION=$2
+
+    for i in ${!packages[@]}; do
+        case ${packages[i]} in
+            yunohost)
+                # Pytest and tests dependencies
+                if ! type "pytest" > /dev/null; then
+                    echo "======================="
+                    echo "> Installing pytest ..."
+                    echo "======================="
+                    apt-get install python-pip
```

Comment:
Ah, actually the check I wanted to add is already there since there's a call to `check_yunohost_vm` at the beginning of the 'test' part. So we're good to go 😄